examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -52,13 +53,13 @@
 )
 
 # Shift the plot origin and add right margin histogram
-fig.shift_origin(yshift="-10.25c", xshift="10.25c")
+fig.shift_origin(yshift=f"-{fig_width + 0.25}c", xshift=f"{fig_width + 0.25}c")
```

Comment:
```suggestion
fig.shift_origin(yshift="-h-0.25c", xshift="w+0.25c")
```

Reply:
Here, **h** is not any more related to the scatter plot but to the upper histogram. To plot at the right side of the scatter plot we have to go back down by the height of the scatter plot not of the upper histogram. Thus, I introduced the variable `fig_width` (I am not 100 % happy with the name). For consistency, I always used `fig_width` here, even partly **h** and **w** can be used directly.